lxd/cgroup/cgroup_cpu.go

```diff
@@ -41,8 +41,8 @@ func ParseCPU(cpuAllowance string, cpuPriority string) (int64, int64, int64, err
 	cpuShares -= int64(10 - cpuPriorityInt)
 
 	// Parse allowance
-	cpuCfsQuota := int64(-1)
-	cpuCfsPeriod := int64(100000)
+	cpuCfsQuota = int64(-1)
+	cpuCfsPeriod = int64(100000)
```

Comment:
Are the type casting `int64()` still needed?

Reply:
Also not needed here! Thanks!